plugins/module_utils/acme/backend_cryptography.py

```diff
@@ -357,6 +361,9 @@ def get_cert_days(self, cert_filename=None, cert_content=None, now=None):
         if cert_content is None:
             return -1
 
+        # Make sure we have at most one PEM. Otherwise cryptography 35.0.0 will barf.
```

Comment:
```suggestion
        # Make sure we have at most one PEM. Otherwise cryptography 36.0.0 will barf.
```
I guess the exact release is still TBD, but certainly not `35.0.0`.